actors/user/src/main/java/org/sunbird/user/util/UserUtil.java

```diff
@@ -585,6 +588,43 @@ public static void toLower(Map<String, Object> userMap) {
               }
             });
   }
+
+  public static String[] userNameGenerator(String name) {
+    if (name == null || name.isEmpty()) return null;
```

Comment:
As per design discussion yesterday, the decision is to generate a user name in following format.

User name (in lowercase without spaces or other special characters) concatenated with a configurable number of digits (default: 4).